css/aboutpage.css

```diff
@@ -0,0 +1,334 @@
+.about-container {
+  /* padding: 0px 20px; */
```

Comment:
- [ ] Please remove all comment pieces of code in all your documents, this will keep your code clean and professional.